lib/process-services-cloud/src/lib/form/mocks/data-table-widget.mock.ts

```diff
@@ -85,6 +85,22 @@ export const mockJsonNestedResponseEuropeCountriesData = {
     }
 };
 
+export const mockJsonNestedResponseEuropeCountriesDataWithSeparatorInPropertyName = {
+    response: {
+        empty: [],
+        'my.data': {
+            'country[data].country': mockEuropeCountriesData
+        },
```

Comment:
What in case of `my:data` instead of `my.data`?

Reply:
path with these special characters was working fine even without that changes (we already have [one test](https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/process-services-cloud/src/lib/form/components/widgets/data-table/data-table.widget.spec.ts#L164) with `-`,  the problem is only with separator which is the `.`.  But just to make sure, i will add one more test with more special characters.

Reply:
Thank you! I'm sorry probably I missed the bit